tools/cross-compile.sh

```diff
@@ -54,8 +54,9 @@ if ./tools/package-deb.sh $VERSION; then
 fi
 create_artefact_generic "linux64"
 
-echo "Linux 64 fully static"
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 make build
+echo "Linux 64 fully static (deprecated)"
+# Only kept for https://github.com/benweissmann/getmic.ro/blob/f90870e948afab8be9ec40884050044b59ed5b7c/index.sh#L197-L204
```

Comment:
Shouldn't we rather ask @benweissmann to update the script?

Reply:
I directly provided the PR: benweissmann/getmic.ro#40
But we've to keep in mind, that this will take effect earliest with the upcoming release.

Reply:
> But we've to keep in mind, that this will take effect earliest with the upcoming release.

Sh*t... so this means that if we remove linux-64 static and merge https://github.com/benweissmann/getmic.ro/pull/40, this Musl check will be broken until the next release?

Reply:
I would suggest doing one release where you publish statically-linked binaries using both the old name and new name (i.e., have the same statically linked binary published both with and without the -static suffix). Then, I can update getmic.ro after that release is published to cut over to the new name seamlessly, and future releases can drop the -static suffix.

Reply:
SGTM.